examples/authentication-example.js

```diff
@@ -37,8 +48,8 @@ var tenantName = "<SEU_TENANT>";
 var scope = "<ESCOPO>";
 
 // Efetuando loginWithKey
-jsonToken = api.authentication.loginWithKey(accessKey, secret, tenantName, scope).then(function(json){
+jsonToken = api.authentication.loginWithKey(accessKey, secret, tenantName, scope).then(function (json) {
```

Comment:
Remover scope, pois o mesmo deixará de fazer parte das APIs
```suggestion
jsonToken = api.authentication.loginWithKey(accessKey, secret, tenantName).then(function (json) {
```